### PR TITLE
PLAT-143536: Added screenshot tests for FlexiblePopupPanles

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Changed the tests from FlexiblePopupPanels to include all the features supported by sampler.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
I added tests for scrimType: translucent and added tests for props: noCloseButton, nextButtonVisibility, prevButtonVisibility. 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143536

### CODE REVIEW

File: tests/screenshot/apps/components/FlexiblePopupPanels.js [hunk 1]
```diff
@@ -79,6 +79,10 @@ const EachPanel = withConfig(
 			title: 'with transparent scrim',
 			component: <FlexPopupPanels open scrimType="transparent" />
 		},
+		{
+			title: 'with transparent scrim',
```
Comment (daniel-stoian-lgp) [inline tests/screenshot/apps/components/FlexiblePopupPanels.js:83]: "translucent"

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: tests/screenshot/apps/components/FlexiblePopupPanels.js [hunk 2]
```diff
@@ -79,6 +79,10 @@ const EachPanel = withConfig(
 			title: 'with transparent scrim',
 			component: <FlexPopupPanels open scrimType="transparent" />
 		},
+		{
+			title: 'with translucent scrim',
+			component: <FlexPopupPanels open scrimType="translucent" />
+		},
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/FlexiblePopupPanels.js:85]: Default value of scrimType is translucent.
This is already being tested in the code 74~77 lines.
Please remove this test.

**Review by dongsuwon — CHANGES_REQUESTED** [review]

File: tests/screenshot/apps/components/FlexiblePopupPanels.js (hunk 2 above)
Comment (alexandrumorariu) [reply tests/screenshot/apps/components/FlexiblePopupPanels.js]: Done.

**Review by dongsuwon — APPROVED** [review]

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**